src/Filetype.js

```diff
@@ -33,9 +41,33 @@ export default class Filetype extends React.Component {
         }
     }
 
+    getAlert() {
+    alert('getAlert from Child');
+  }
+
+
+    check(event){
+        console.log("filtype "+this.props.filetype);
+        console.log("Les formats",this.props.formats);
+        if(this.props.disabled===false){
+      this.setState({
```

Comment:
Strange indentation.
Please use spaces to indent (this is environment-independant).